src/PricingManager/Rule/Dao.php

```diff
@@ -76,11 +76,19 @@ public function create(): void
      */
     public function save(): void
     {
-        if (!$this->model->getId()) {
-            $this->create();
-        }
+        try {
+            $this->db->beginTransaction();
+            if (!$this->model->getId()) {
+                $this->create();
+            }
 
-        $this->update();
+            $this->update();
+            $this->db->commit();
+        } catch (\Exception $e) {
```

Comment:
Can you please import the Exception instead of using the global namespace?

Reply:
Hmm... but it is like this everywhere in the bundle, so I would keep it like this.
![image](https://github.com/user-attachments/assets/0b951eec-aea3-4627-8620-a0114c841150)
If you want it differently - feel free to change it, but I would then suggest to also change it in the other places to be consistent.

BR

Reply:
I know, but improving bit by bit also works :) 
In other bundles we already have CS Fixer Rules in place, but it's only a small change so I will do it quickly :)